bzt/engine.py

```diff
@@ -494,7 +494,6 @@ def __prepare_aggregator(self):
         cls = self.config.get(SETTINGS).get("aggregator", "")
         if not cls:
             self.log.warning("Proceeding without aggregator, no results analysis")
-            self.aggregator = EngineModule()
```

Comment:
This works because we instantiate `engine.aggregator` to be an instance of `Aggregator` already in the [constructor](https://github.com/Blazemeter/taurus/pull/500/files#diff-fc553025b5166711abd2a77212a6f9e5L74).